docs/redoc-bundle/src/main/scala/sttp/tapir/redoc/bundle/RedocInterpreter.scala

```diff
@@ -1,86 +1,102 @@
-  ): List[ServerEndpoint[Any, F]] =
-    fromEndpoints(endpoints.map(_.endpoint), info)
+  ): List[ServerEndpoint[Any, F]] = fromEndpoints(endpoints.map(_.endpoint), info, Nil)
 
   def fromServerEndpoints[F[_]](
       endpoints: List[ServerEndpoint[_, F]],
       title: String,
       version: String
   ): List[ServerEndpoint[Any, F]] =
-    fromEndpoints(endpoints.map(_.endpoint), Info(title, version))
+    fromEndpoints(endpoints.map(_.endpoint), Info(title, version), Nil)
 }
 
 object RedocInterpreter {
 
   /** Allows interpreting lists of [[sttp.tapir.Endpoint]]s or [[ServerEndpoint]]s as Redoc docs. The docs will be serialised in the yaml
     * format, and will be available using the configured `prefix` path, by default `/docs`.
     *
-    * Usage: pass the result of `SwaggerInterpreter().fromEndpoints[F]` to your server interpreter.
+    * Usage: pass the result of `RedocInterpreter().fromEndpoints[F]` to your server interpreter.
     *
-    * @param docsExtensions
-    *   The top-level documentation extensions to be included in the generated OpenAPI docs.
-    * @param docsOptions
+    * @param openAPIInterpreterOptions
     *   The options that will be passed to the [[OpenAPIDocsInterpreter]].
-    * @param prefix
-    *   The path prefix from which the documentation will be served, as a list of path segments. Defaults to `List(docs)`, so the address of
-    *   the docs will be `/docs`.
-    * @param yamlName
-    *   The name of the file, through which the yaml documentation will be served. Defaults to `docs.yaml`.
+    * @param customiseDocsModel
+    *   A function which can customise the [[OpenAPI]] model created by the [[OpenAPIDocsInterpreter]] for the given endpoints, allowing
+    *   adding e.g. server information.
+    * @param redocUIOptions
+    *   Options passed to [[Redoc]] to customise how the documentation is exposed, e.g. the path.
+    * @param addServerWhenContextPathPresent
+    *   Should a default server entry be added to the generated [[OpenAPI]] model pointing to the context path, if a non-empty context path
+    *   is specified in `swaggerUIOptions`. In presence of a context path, either the endpoints need to be prefixed with the context path,
+    *   or a server entry must be added, for invocations from within the Swagger UI to work properly. Defaults to `true`.
+    * @param redocVersion
+    *   \- version of Redoc library
     */
   def apply(
-      docsExtensions: List[DocsExtension[_]] = Nil,
-      docsOptions: OpenAPIDocsOptions = OpenAPIDocsOptions.default,
-      prefix: List[String] = List("docs"),
-      yamlName: String = "docs.yaml",
-      htmlName: String = "index.html",
+      openAPIInterpreterOptions: OpenAPIDocsOptions = OpenAPIDocsOptions.default,
+      customiseDocsModel: OpenAPI => OpenAPI = identity,
+      redocUIOptions: RedocUIOptions = RedocUIOptions.default,
+      addServerWhenContextPathPresent: Boolean = true,
       redocVersion: String = "2.0.0-rc.56"
```

Comment:
maybe this should be part of `RedocUIOptions`?

Reply:
Sure it's a good candidate :) I will move it also.